src/com/jambit/onboarding2020/tbrpg/domain/Player/Player.java

```diff
@@ -38,8 +38,65 @@ public void sell(Item item) {
         this.balance += item.getSellValue();
     }
 
-    public void attack (Enemy damagedPerson) throws EnemyDeadException {
-        damagedPerson.decreaseHealthState(this.attackDamage);
+    public void attack (Enemy damagedPerson, Weapon equippedWeapon) throws EnemyDeadException {
```

Comment:
Man muss hier nicht equippedWeapon übergebn, da du einfach in der Methode ds Weapon abfragen kannst